packages/config/.eslintrc.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
```

Comment:
You can remove this file and the package will use the default `.eslintrc.js` from the root folder.